satpy/etc/readers/hdfeos_mod35.yaml

```diff
@@ -0,0 +1,152 @@
+reader:
+  default_datasets: [cloud_mask]
+  description: MODIS HDF-EOS Cloud Mask Reader
+  name: hdfeos_mod35
+  reader: !!python/name:satpy.readers.yaml_reader.FileYAMLReader
+  sensors: [modis]
+
+navigations:
+  hdf_eos_geo:
+      description: MODIS navigation
+      file_type: hdf_eos_geo
+      latitude_key: Latitude
+      longitude_key: Longitude
+      nadir_resolution: [1000]
+      rows_per_scan: 10
+
+datasets:
+  cloud_mask_1k:
+    name:
+    #bits: [num_byte, start_bit, num_bits]
+    #Byte 1
+      cloud_mask: {bits: [0,0,1]}
+      fov_quality: {bits: [0,1,2]}
+      day_night_path: {bits: [0,3,1]}
```

Comment:
I hadn't thought of putting all the datasets together in this fashion before, but I'm fine with it of satpy's `available_dataset_names` works with it.

Reply:
Yeah haven't thought about that. It is kind of s "missuse" I guess but otherwise there would be much repitition. I will test it and am also willing to change it if some nice idea pops up.